sklearn/datasets/tests/test_samples_generator.py

```diff
@@ -112,7 +112,7 @@ def test_make_classification_informative_features():
         (2, [1 / 2] * 2, 2),
         (2, [3 / 4, 1 / 4], 2),
         (10, [1 / 3] * 3, 10),
-        (int(64), [1], 1),
+        ((64), [1], 1),
```

Comment:
```suggestion
        (64, [1], 1),
```
?

Reply:
Indeed, but these are automatic fixes and neither `ruff` nor `black` seem to be catching this. But then see my comment https://github.com/scikit-learn/scikit-learn/pull/30976#discussion_r2001143740. I suggest deferring pre-commit fixes to a different PR any way.